template.rb

```diff
@@ -100,6 +103,10 @@ def delete_test_folder
   FileUtils.rm_rf('test')
 end
 
+def install_addon_prompt(addon)
+  "Would you like to add #{addon} addon? [Yn]"
```

Comment:
Add the article `the` for the specific addon.

```suggestion
  "Would you like to add the #{addon} addon? [Yn]"
```